pyquil/tests/test_api.py

```diff
@@ -27,22 +28,37 @@
 from pyquil.api.qpu import append_measures_to_program
 from pyquil.quil import Program
 from pyquil.paulis import PauliTerm
-from pyquil.gates import CNOT, H, MEASURE, PHASE, Z
+from pyquil.gates import CNOT, H, MEASURE, PHASE, Z, RZ, RX, CZ
 from pyquil.device import ISA
+from pyquil.quilbase import Pragma
 
 EMPTY_PROGRAM = Program()
 BELL_STATE = Program(H(0), CNOT(0, 1))
 BELL_STATE_MEASURE = Program(H(0), CNOT(0, 1), MEASURE(0, 0), MEASURE(1, 1))
+COMPILED_BELL_STATE = Program([
+    Pragma("EXPECTED_REWIRING", ('"#(0 1 2 3)"',)),
```

Comment:
This feels a little brittle. I'm not actually sure what the `expected_rewirings` are currently used for, naively I feel like we should just ignore them in tests. But that's my two-second-thought.

Reply:
I agree, but this is what the test was supposed to be doing as originally written